packages/react-tabster/e2e/useKeyBorg.e2e.ts

```diff
@@ -0,0 +1,10 @@
+import { KEYBOARD_NAV_ATTRIBUTE } from '../src/symbols';
+
+describe('Default', () => {
+  beforeEach(() => cy.visitStory('tabster', 'Default'));
+
+  it('should open when clicked', () => {
+    cy.contains('Start').focus().realPress('Tab');
+    cy.contains('Finish').should('be.focused').parent().should('have.attr', KEYBOARD_NAV_ATTRIBUTE);
+  });
+});
```

Comment:
nit: this file should be named `useKeyborg.e2e.ts` as a hook is named `useKeyborg`